pkg/graph/resolvers/mto_milestone_test.go

```diff
@@ -516,3 +522,65 @@ func (suite *ResolverSuite) TestMTOMilestoneUpdateLinkedSolutions_UnlinkByCommon
 	suite.NoError(err)
 	suite.Len(linkedSolutions, 0)
 }
+
+func (suite *ResolverSuite) TestMTOMilestoneNoLinkedSolutions_SinglePlan() {
```

Comment:
This probably doesn't hurt, but I'm not sure you need it if you have multiple test cases in the other loader. Perhaps it's worthwhile to test a case when no results will be returned by SQL at all?